migration/migrator/data/course_tables.sql

```diff
@@ -714,7 +714,11 @@ CREATE TABLE public.course_materials (
     hidden_from_students boolean,
     priority double precision NOT NULL,
     url text,
-    title character varying(255)
+    title character varying(255),
+    uploaded_by character varying(255) DEFAULT NULL::character varying,
+    upload_date timestamp with time zone,
```

Comment:
![스크린샷 2024-06-11 오전 10 22 15](https://github.com/Submitty/Submitty/assets/123261952/ef384f75-40ca-499a-bd22-8490d19c87cb)

I am getting frog robot when I visit http://localhost:1511/courses/s24/sample/course_materials
I did try submitty_install & submitty_install clean
Could you let me know if you can reproduce the issue?

Reply:
Let me reinstall it and test it out.

Reply:
Fixed.